### PR TITLE
Scene: `msaaSamples` remove `readonly`

### PR DESCRIPTION
**Pull request by ShenWeiQun**
In `Scene` of `msaaSamples` is can set.
```js
  msaaSamples: {
    get: function () {
      return this._msaaSamples;
    },
    set: function (value) {
      value = Math.min(value, ContextLimits.maximumSamples);
      this._msaaSamples = value;
    },
  },
```
But in `Cesium.d.ts` will `Throws:DeveloperError `
Cesium.d.ts
```js
    /**
     * The sample rate of multisample antialiasing (values greater than 1 enable MSAA).
     */
    readonly msaaSamples: number;
```

I think that is not a good type description.

So:
```diff
- * @readonly
```

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ShenWeiQun!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/main/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/CesiumGS/cesium/tree/main/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.

**Comment by ggetz** [comment]
Thanks @ShenWeiQun!

@ebogo1 Could you please review?

File: CHANGES.md [hunk 1]
```diff
@@ -16,6 +16,7 @@
 
 ##### Fixes :wrench:
 
+- Fixed `Scene` documentation for `msaaSamples` changed to not `readonly`. [#10205](https://github.com/CesiumGS/cesium/pull/10205)
```
Comment (ebogo1) [inline CHANGES.md:19]:
```suggestion
- Fixed `Scene` documentation for `msaaSamples` property. [#10205](https://github.com/CesiumGS/cesium/pull/10205)
```

**Review by ebogo1 — COMMENTED** [review]
Thanks for catching this @ShenWeiQun! I had a small suggestion for the CHANGES.md wording but otherwise this is good to go.

**Comment by ShenWeiQun** [comment]
@ebogo1  Thank you for your correction

**Pull request merged**